package.json

```diff
@@ -5,7 +5,9 @@
   "repository": "https://github.com/concourse/concourse",
   "license": "Apache-2.0",
   "dependencies": {
-    "request": "^2.86.0"
+    "element-ui": "^2.15.0",
+    "request": "^2.86.0",
+    "vue-resource": "^1.5.1"
```

Comment:
is this change required for the PR?

Reply:
No, I don't think so. It came about because I needed the UI to run the binary locally, but it also works without it.

Reply:
More precisely, I followed the instructions [here](https://discuss.concourse-ci.org/t/error-in-dev-environment/992/2), since initially I was getting the same error message about main.css. But I tested the binary without these parameters too, it works fine.

Reply:
Could you revert the change? Those instructions aren't accurate - we don't use any those new dependencies (we don't even use Vue). I think the solution to the error message is just to run `yarn build`

Reply:
Done